### ISSUE TITLE
PEP checks headers in different ways

### ISSUE THREAD

**Issue by jcalderin** [telefonicaid/fiware-orion-pep#110]
When the header doesnt exist, PEP do the following checks 

 header  | check 
---------- | ---------
Accept | Not Checked, go to AC 
content-type | checked in the first pep petition 
Fiware-service | checked in the first pep petition
Fiware-Servicepath | checked in the "projects" petition to Keystone

The functionality is OK, but is a good enhancement do all checks in the same way

# Log Accept
```
time=2015-01-05T13:03:06.725Z | lvl=DEBUG | corr=489d5d85-69c1-44ac-a3c9-2ee020ce77fe | trans=489d5d85-69c1-44ac-a3c9-2ee020ce77fe | op=/v1/updateContext | msg=Sending access request:
{
    "uri": "http://192.168.1.37:8082/pdp/v3",
    "method": "POST",
    "headers": {
        "Content-Type": "application/xml",
        "Accept": "application/xml",
        "fiware-service": "atlantic",
        "fiware-servicepath": "/"
    },
    "body": "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Request xmlns=\"urn:oasis:names:tc:xacml:3.0:core:schema:wd-17\"\n         xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n         xsi:schemaLocation=\"urn:oasis:names:tc:xacml:3.0:core:schema:wd-17 http://docs.oasis-open.org/xacml/3.0/xacml-core-v3-schema-wd-17.xsd\"\n         ReturnPolicyIdList=\"false\"\n         CombinedDecision=\"false\">\n<!-- user roles identifiers-->\n<Attributes Category=\"urn:oasis:names:tc:xacml:1.0:subject-category:access-subject\">\n    <Attribute IncludeInResult=\"false\"\n           AttributeId=\"urn:oasis:names:tc:xacml:1.0:subject:subject-id\">\n    <AttributeValue\n            DataType=\"http://www.w3.org/2001/XMLSchema#string\"\n            >33be1f92b96f45f69dd904eecf3098fd</AttributeValue>\n</Attribute>\n</Attributes>\n<!-- fiware resource name being accessed: organization id -->\n<Attributes\n        Category=\"urn:oasis:names:tc:xacml:3.0:attribute-category:resource\">\n    <Attribute IncludeInResult=\"false\"\n               AttributeId=\"urn:oasis:names:tc:xacml:1.0:resource:resource-id\">\n        <AttributeValue DataType=\"http://www.w3.org/2001/XMLSchema#string\"\n                >fiware:orion:atlantic:/::</AttributeValue>\n    </Attribute>\n</Attributes>\n<!-- action performed -->\n<Attributes\n        Category=\"urn:oasis:names:tc:xacml:3.0:attribute-category:action\">\n    <Attribute IncludeInResult=\"false\"\n               AttributeId=\"urn:oasis:names:tc:xacml:1.0:action:action-id\">\n        <AttributeValue DataType=\"http://www.w3.org/2001/XMLSchema#string\"\n                >create</AttributeValue>\n    </Attribute>\n</Attributes>\n</Request>"
}

time=2015-01-05T13:03:06.744Z | lvl=DEBUG | corr=489d5d85-69c1-44ac-a3c9-2ee020ce77fe | trans=489d5d85-69c1-44ac-a3c9-2ee020ce77fe | op=/v1/updateContext | msg=Request access denied
time=2015-01-05T13:03:06.744Z | lvl=DEBUG | corr=489d5d85-69c1-44ac-a3c9-2ee020ce77fe | trans=489d5d85-69c1-44ac-a3c9-2ee020ce77fe | op=/v1/updateContext | msg=response-time: 67 statusCode: 403
```

# Log content-type
```
time=2015-01-05T13:04:23.757Z | lvl=ERROR | corr=94e838f2-d649-4c80-91b7-22e18d29ff9d | trans=94e838f2-d649-4c80-91b7-22e18d29ff9d | op=/v1/updateContext | msg=[ORION-PLUGIN-004] Unknown content type: undefined
time=2015-01-05T13:04:23.764Z | lvl=DEBUG | corr=94e838f2-d649-4c80-91b7-22e18d29ff9d | trans=94e838f2-d649-4c80-91b7-22e18d29ff9d | op=/v1/updateContext | msg=response-time: 11 statusCode: 403
```

# Fiware-Service
```
time=2015-01-05T13:00:59.834Z | lvl=ERROR | corr=8b003fd4-6972-4330-a55a-c7da513a36d9 | trans=8b003fd4-6972-4330-a55a-c7da513a36d9 | op=/v1/updateContext | msg=[PROXY-GEN-001] Organization headers not found
time=2015-01-05T13:00:59.835Z | lvl=DEBUG | corr=8b003fd4-6972-4330-a55a-c7da513a36d9 | trans=8b003fd4-6972-4330-a55a-c7da513a36d9 | op=/v1/updateContext | msg=response-time: 1 statusCode: 403
```

# Fiware-Servicepath
```
time=2015-01-05T13:00:59.306Z | lvl=ERROR | corr=68312705-2ae7-46f1-82f3-5d0038f4b309 | trans=68312705-2ae7-46f1-82f3-5d0038f4b309 | op=/v1/updateContext | msg=Couldn't find subservice id in Keystone with name undefined
time=2015-01-05T13:00:59.310Z | lvl=DEBUG | corr=68312705-2ae7-46f1-82f3-5d0038f4b309 | trans=68312705-2ae7-46f1-82f3-5d0038f4b309 | op=/v1/updateContext | msg=response-time: 403 statusCode: 403

```

**Comment by jcalderin** [comment]
In the "content-type" content the functionality is different and works as the "Accept", getting to AC

**Comment by fgalan** [comment]
Implemented in PR #167 (for the records).
